metaphor/snowflake/extractor.py

```diff
@@ -195,7 +195,7 @@ def _init_dataset(
         if row_count:
             dataset.statistics.record_count = float(row_count)
         if table_bytes:
-            dataset.statistics.data_size = table_bytes / 1048576
+            dataset.statistics.data_size = table_bytes / 1000000
```

Comment:
Consider this for clarify

```suggestion
            dataset.statistics.data_size = table_bytes / (1000*1000)  # in MB
```